src/me/StevenLawson/TotalFreedomMod/Commands/Command_tagclean.java

```diff
@@ -0,0 +1,52 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import me.StevenLawson.TotalFreedomMod.TFM_PlayerData;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Remove forbiiden colors, things from Tags.", usage = "/<command>", aliases = "tc")
+public class Command_tagclean extends TFM_Command
+{
+    private static final ChatColor[] BLOCKED = new ChatColor[]
+    {
+        ChatColor.MAGIC,
+        ChatColor.STRIKETHROUGH,
+        ChatColor.ITALIC,
+        ChatColor.UNDERLINE,
+        ChatColor.BLACK
+    };
```

Comment:
I am referring to the ; It doesnt look like it should be there, I am 99% sure that couldnt compile.